lib/widgets/packs/announcement_banner_widget.dart

```diff
@@ -78,19 +79,17 @@ class AnnouncementBannerState extends State<AnnouncementBanner>
                       children: [
                         buildSpacer(),
                         Padding(
-                          padding: const EdgeInsets.only(bottom: 8.0),
-                          child: MaterialBanner(
-                            backgroundColor: MeditoColors.darkMoon,
-                            content: _buildTextColumn(snapshot, context),
-                            leading: snapshot.data.icon.isNotEmptyAndNotNull()
-                                ? buildCircleAvatar(snapshot)
-                                : Container(),
-                            actions: [
-                              _buildDismissButton(),
-                              _buildPositiveButton(snapshot),
-                            ],
-                          ),
-                        ),
+                            padding:
+                                const EdgeInsets.only(top: 20.0, bottom: 10.0),
+                            child: MaterialBanner(
+                                backgroundColor: MeditoColors.darkMoon,
+                                content: _buildTextAndButtonColumn(
+                                    snapshot, context),
+                                leading:
+                                    snapshot.data.icon.isNotEmptyAndNotNull()
+                                        ? buildCircleAvatar(snapshot)
+                                        : Container(),
+                                actions: [Container()])),
```

Comment:
You don't wanna use this `actions` anymore?

Reply:
The thing is, if I use `actions`, and when the main CTA is empty, it aligns the text and the dismiss button side by side.

Reply:
https://github.com/meditohq/medito-app/issues/165#issuecomment-913371196

Reply:
Oh yeah super weird!